encoder_res: 64.0
gear_reduction: 43.8
serial_port_name: "/dev/ttyACM0"
